### DISCUSSION TITLE
Getting device logs in storage but not injecting any data in database

### DISCUSSION THREAD

**Discussion by amrkamboj1988** [eusonlito/GPS-Tracker#73] [Q&A] [answered]
First of all, it's very nice script that is helping lot of developers. Thanks for sharing you knowledge.

I have installed it successfully and created a server on port 8840 (on which I will be listening devices). Added few  devices.

After setup i can see logs appending in following files:

> /storage/logs/server/{year}/{month}/{date}/{port}.log
> /storage/logs/server/{year}/{month}/{date}/{port}-debug.log
> /storage/logs/server/{year}/{month}/{date}/{port}-connection.log

but When i am checking devices details there is no connection and no trips data 

![brave_ujtKIvtYvH](https://github.com/user-attachments/assets/3b8c34da-6480-4b7b-a1d5-eabd3035a235)
![brave_UjtARdVbAU](https://github.com/user-attachments/assets/993f6203-3db5-4c03-bea6-36df34007ca3)
![brave_MjfugEgf00](https://github.com/user-attachments/assets/0a49c71e-215e-4772-81f4-0ae51ba065fb)

I have added IP in devices - SERVER,0,13.55.41.202,8840,0#

Please guide if I am missing something

**Comment by eusonlito** [comment]
What is your device brand and model? Can you paste/upload here the debug log?

**Comment by amrkamboj1988** [comment]
Brand is from [www.libitechnologies.com](url) and device is MT100. This  device supports multiple IPs and protocols. Currecntly device is working on Concox GT06 Protocol.

[MT100 User Mannual.pdf](https://github.com/user-attachments/files/16813164/MT100.User.Mannual.pdf)
[8840.log](https://github.com/user-attachments/files/16812412/8840.log)
[8840-connection.log](https://github.com/user-attachments/files/16812414/8840-connection.log)
[8840-debug.log](https://github.com/user-attachments/files/16812415/8840-debug.log)

**Reply by eusonlito** [reply]
Your device serial is `0862209128004587`?

**Reply by amrkamboj1988** [reply]
yes, That is one put of 3 devices

![image](https://github.com/user-attachments/assets/e9cdc604-505c-410c-b70e-37094fe4ddb0)

**Reply by eusonlito** [reply, marked as answer]
And is configured with the initial zero as device serial on this gps platform?

**Reply by amrkamboj1988** [reply]
Crap, that is very crazy. it's sending 0 as extra digit with imei number. I decoded the login string and sending back ack while working on Node script but now I got to know that node script didn't work and device didn't sent location back after ack because of this silly 0.
Thanks for your guidance. I'll write you an email for the paid service if you can offer.
Thanks again.
